english/take-nap/cucumber-induced-nap.md

```diff
@@ -0,0 +1,4 @@
+You unpack your porta-bed that you got from [Costco](http://www.costco.com) 
+And lay your head on the [MyPillow](http://www.mypillow.com) that you bought 
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.